.ci/pitest.sh

```diff
@@ -154,6 +154,8 @@ pitest-imports)
   "ImportOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>        if (isStatic) {</span></pre></td></tr>"
   "ImportOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>        return !beforeFirstImport &#38;&#38; line - lastImportLine &#62; 1;</span></pre></td></tr>"
   "ImportOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>            staticImportSeparator = isStatic &#38;&#38; separated;</span></pre></td></tr>"
+  "ImportOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>        else if (option == ImportOrderOption.INFLOW) {</span></pre></td></tr>"
+  "ImportOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>        if (option == ImportOrderOption.TOP || option == ImportOrderOption.ABOVE) {</span></pre></td></tr>"
```

Comment:
These cases were relying on the IllegalStateException before. Now that that code is gone they are surviving.